NAMESPACE

```diff
@@ -1,3 +1,23 @@
 # Generated by roxygen2: do not edit by hand
 
-export(read_stuff)
+export(add_to_dictionary)
+export(add_to_report)
+export(check_date_sequence)
+export(check_subject_ids)
+export(clean_data)
+export(clean_using_dictionary)
+export(convert_numeric_to_date)
+export(convert_to_numeric)
+export(correct_subject_ids)
+export(find_duplicates)
+export(print_report)
+export(remove_constant)
+export(remove_duplicates)
+export(replace_missing_values)
+export(scan_data)
+export(span)
+export(standardize_column_names)
+export(standardize_dates)
```

Comment:
Most of the function names are written in plural except:
* `check_date_sequence, convert_numeric_to_date, convert_to_numeric`: I don't think it will make sense to apply a plural on these function names given the context and the action that we want to reflect on the function names.
*  `span, scan_data, add_to_dictionary, add_to_report`: I think these names can remain as they are currently given that we only have 1 input data, 1 dictionary and 1 report.
* `remove_constant`: has now been renamed to `remove_constants`.